third_party/purge/upd.purge.php

```diff
@@ -0,0 +1,98 @@
+<?php  if ( ! defined('BASEPATH')) exit('No direct script access allowed');
+
+
+
+class Purge_upd {
+	
+	public $version = '1.0';
```

Comment:
This change was proposed to be 1.0.5, correct? Though it may be considered a 1.1, it's quite a change.